lms/static/js/spec/views/fields_spec.js

```diff
@@ -234,8 +234,8 @@ define(['backbone', 'jquery', 'underscore', 'common/js/spec_helpers/ajax_helpers
                     title: 'About me',
                     valueAttribute: 'bio',
                     helpMessage: 'Wicked is good',
-                    placeholderValue: "Tell other edX learners a little about yourself: where you live, " +
-                        "what your interests are, why you’re taking courses on edX, or what you hope to learn.",
```

Comment:
Can you replace "edX" with {platform_name}, as we do elsewhere throughout the platform?